packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -327,39 +327,58 @@ skipInNode(describe)('plugin-meeting', () => {
       }));
 
     it('alice update Audio', () => alice.meeting.getMediaStreams({sendAudio: true})
-      .then((response) => alice.meeting.updateAudio({
-        sendAudio: true,
-        receiveAudio: true,
-        stream: response[0]
-      })
-        .then(() => {
-          console.log('AUDIO ', alice.meeting.mediaProperties.peerConnection.audioTransceiver.sender.track);
-          assert.equal(alice.meeting.mediaProperties.audioTrack.id, response[0].getAudioTracks()[0].id);
-        })));
+      .then((response) => Promise.all([
+        testUtils.delayedPromise(alice.meeting.updateAudio({
+          sendAudio: true,
+          receiveAudio: true,
+          stream: response[0]
+        })
+          .then(() => {
+            console.log('AUDIO ', alice.meeting.mediaProperties.peerConnection.audioTransceiver.sender.track);
+            assert.equal(alice.meeting.mediaProperties.audioTrack.id, response[0].getAudioTracks()[0].id);
+          })),
+        testUtils.waitForEvents([{scope: alice.meeting, event: 'media:ready'}])
+          .then((response) => {
+            console.log('MEDIA:READY event ', response[0].result);
+            assert.equal(response[0].result.type === 'local', true);
+          })
+      ])));
 
     it('alice updadte video', () => alice.meeting.getMediaStreams({sendVideo: true})
```

Comment:
Not necessary, but `updadte` isn't a word 😆

Reply:
@adamweeks  babies user that word